2.3v2.c

```diff
@@ -0,0 +1,53 @@
+#include <stdio.h>
+
+
+//�������� ���������� ��������� � ��������.
+
+
+#define range 2001

```

Comment:
```suggestion
#define RANGE 2001
```
define константы принято писать капсом и разделять слова с помощью `_`.